packages/ai/src/chains/copywriter/chain.ts

```diff
@@ -70,7 +70,7 @@ export const createChain = <ModelMessageFormat>(): Chain<
         id: name,
         ...createErrorResponse({
           status: 404,
-          error: "ai.parseError",
```

Comment:
seems like you are chaning "error codes" to "error message" ... but:
1. does it mean you will have to parse it instead of comparing string?
2. why is it not changed everywhere then? other places still use the error code

Reply:
I haven't touched this except message. Not sure about what you are writing.